config/locales/en.yml

```diff
@@ -39,6 +39,10 @@ en:
         restricted_access_text: 'Search inside at HathiTrust'
         etas_text: 'Check out digital copy through HathiTrust'
         etas_additional_text: 'By special arrangement in response to the COVID-19 pandemic, patrons may access this resource online through the HathiTrust Emergency Temporary Access Service.'
+      facet_tooltips:
+        access_facet:
+          free_to_read: 'These materials don''t require a Penn State login'
```

Comment:
```suggestion
          free_to_read: 'These materials don't require a Penn State login'
```

Reply:
@banukutlu - the double apostrophe (`''`) was intentional - it is an escape for the single apostrophe because this text is in single quotes. Alternatives would be to use double quotes or no quotes:

```
free_to_read: These materials don't require a Penn State login
free_to_read: "These materials don't require a Penn State login"
```

Do you have a preference?

Reply:
Ahh my bad sorry!  What you had is great, I rather single apostrophe here as well, no need change 👌

Reply:
For future reference, the quotes aren't always necessary in yaml, but if it needs them, you can use double-quotes. If you happen to need double-quotes in an already double-quoted string, then `"He said \"What!?\""`